tests/functional/test_freeze.py

```diff
@@ -258,41 +259,31 @@ def test_freeze_mercurial_clone(script, tmpdir):
     _check_output(result, expected)
 
 
-@pytest.mark.network
 def test_freeze_bazaar_clone(script, tmpdir):
     """
     Test freezing a Bazaar clone.
 
     """
-
-    checkout_path = local_checkout(
-        'bzr+http://bazaar.launchpad.net/%7Edjango-wikiapp/django-wikiapp/'
-        'release-0.1',
-        tmpdir.join("cache"),
-    )
-    # bzr internally stores windows drives as uppercase; we'll match that.
+    try:
+        checkout_path = _create_test_package(script, vcs='bazaar')
+    except OSError as e:
+        pytest.skip('Invoking `bzr` failed: %s' % e)
```

Comment:
@msabramo Not sure there's a specific "make the tests work on Windows" issue, just a general acceptance that they never have worked very well. They run very slowly too, so I've never even bothered trying to collect the output to see what's wrong - it could possibly be pretty shallow (but I suspect not :-()

Without a CI service to run the tests on, it's also too easy to fix them and then have them break again. And I don't think Appveyor is fast enough to work for us. Worth trying, but I'm not hopeful.

Basically a big chunk of work no-one's got round to yet, nothing more than that.